scripts/ai/mcp/mcp_memory_wrapper.sh

```
@@ -40,12 +40,24 @@ esac
+  MEMORY_PYTHON="${REPO_ROOT}/.venv/bin/python"
+elif command -v python3 >/dev/null 2>&1; then
+  MEMORY_PYTHON="$(command -v python3)"
+else
+  printf '%s\n' "Memory MCP requires a project Python interpreter" >&2
+  exit 69
+fi
+
+export PYTHONPATH="${REPO_ROOT}/src${PYTHONPATH:+:${PYTHONPATH}}"
 if [[ -z "${MEMORY_FILE_PATH:-}" ]]; then
-  export PYTHONPATH="${REPO_ROOT}/src${PYTHONPATH:+:${PYTHONPATH}}"
   export MEMORY_FILE_PATH="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)